src/wallets/injected/SenderWallet.ts

```diff
@@ -56,7 +56,7 @@ class SenderWallet implements InjectedWallet {
   };
 
   private isInstalled = () => {
-    return !!window.wallet;
+    return typeof window.near !== 'undefined' && window.near.isSender;
```

Comment:
We could be more terse on this expression with:
```ts
return !!window.near?.isSender;
```

Reply:
sounds good